composer.json

```diff
@@ -14,7 +14,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.3",
```

Comment:
Why is this being changed?

Reply:
To make my IDE stop freaking out. I won't commit that